app/src/test/java/se/inera/intyg/minaintyg/util/html/CertificateQuestionValueHTMLFactoryTest.java

```diff
@@ -52,4 +54,27 @@ void shouldReturnHTMLForTable() {
         "<table className=\"ids-table\"><thead><th>heading 1</th><th>heading 2</th></thead><tbody><tr><td>Value 1</td><td>Value 2</td></tr></tbody></table>",
         result);
   }
+
+  @Test
+  void shouldReturnHTMLForItemList() {
+    final var value = CertificateQuestionValueItemList
+        .builder()
+        .values(List.of(
+            CertificationQuestionValueItem.builder()
+                .label("label 1")
+                .value("value 1")
+                .build(),
+            CertificationQuestionValueItem.builder()
+                .label("label 2")
+                .value("value 2")
+                .build()
+        ))
+        .build();
+
+    final var result = CertificateQuestionValueHTMLFactory.itemList(value);
+
+    assertEquals(
+        "<p><h5 className=\"ids-heading-5\">label 1</h5><p>value 1</p><h5 className=\"ids-heading-5\">label 2</h5><p>value 2</p></p>",
```

Comment:
From the example HTML I don't think it was a <p> around the question

Reply:
Fixed!